File: src/java/BubbleSort.java
// Bubble sort in Java

import java.util.*;

class Main {

static void bubbleSort(int array[],int size) {
    for (int i = 0; i < size- 1 ; i++)    {
      for (int j = 0; j < size - i- 1 ; j++)     {
        if (array[j] > array[j + 1]) {
            // swapping a[j+1] and a[j]
          int temp = array[j];
          array[j] = array[j + 1];
          array[j + 1] = temp;
        }
      }
    }
  }
  static void printArray(int array[],int n)
    {
        for (int i = 0; i < n; i++)
            System.out.print(array[i] + " "); 

        System.out.println();
    }

  public static void main(String args[]) {
    Scanner sc = new Scanner(System.in);  
    int n;
    System.out.println("Enter the size of list:");
    n=sc.nextInt();
    int[] list = new int[n];
    System.out.println("Enter the elements to sort: ");
    for(int i=0;i<n;i++)      {
        list[i]=sc.nextInt();
    }
    System.out.println("Elements in sorted order:");
    bubbleSort(list,n);
    printArray(list,n);
  }
}

File: src/python/bubble_sort.py
# bubble sort in python

def bubble_sort(arr):
    n = len(arr)
    for i in range(1, n):
        for j in range(0, n-1):
            if arr[j] > arr[j+1]:
                temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp

    return arr

  
unsorted_arr = [3, 6, 7, 1, 5, 9, 2, 0, 4, 8]
sorted_arr = bubble_sort(unsorted_arr)
print(sorted_arr)
